models/pull.js

```diff
@@ -75,6 +75,9 @@ Pull.prototype.update = function() {
 };
 
 Pull.prototype.syncToIssue = function() {
+   return Promise.resolve(this);
```

Comment:
Nope, [`Promise.resolve()`](https://www.npmjs.com/package/promise#promiseresolvevalue) is a static function, not a thing you should instantiate.